data.sql

```diff
@@ -4,3 +4,11 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', DATE '2018-11-15', 2, TRUE, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', DATE '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', DATE '2017-05-12', 5, TRUE, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', DATE '2020-02-08', 0, FALSE, 11);
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=2020%2C%20and%20currently-,weighs%20%2D11kg,-.%20He%20is%20not), Please add the correct value of the weight here in this insertion because it's supposed to be `-11` instead of `11`.